### DISCUSSION TITLE
Make Haliscan Brimmed Hat transmoggable by all

### DISCUSSION THREAD

**Discussion by siorse** [chromiecraft/chromiecraft#5724] [Suggestions]
This is a Nice Hat. It's from a quest from a Cool Guy outside of Tanaris.

It's a very very nice hat. I travelled all the way from Tanaris to Stormwind to try and transmog it.

Only to find that I could not! For it was cloth and I wear mail.

My suggestion is that this Nice Hat becomes transmoggable for all who have completed this quest. 

It would make people happy. And I think that is important.

Thank you for your consideration.

Siorse

**Comment by cachocarne** [comment]
Yay, let's start a Mariachi Guild.
To be honest, I though Transmogrification should be ALL THE WAY so everybody can xmog anything or NOT AT ALL.

**Reply by siorse** [reply]
It should. And we should start with this hat, as I think it is the most vital piece of unachieved transmog so far in game.

**Comment by Iramtempus** [comment]
Seconded

**Reply by siorse** [reply]
Thank you for your support on this endeavour

**Comment by PaullyDM** [comment]
I also agree, let us transmog with any piece of gear, no matter if its from Plate to Cloth, etc. :D
